src/services/router.go

```diff
@@ -139,17 +139,17 @@ func initBooksRoute() {
 	router.GET("categories/:cat_id/books", bookService.getBooks)
 	//get all books
 	router.GET("/books", bookService.showAllBooks)
+	//get five most popular books
 	router.GET("/mostPopularBooks", bookService.FiveMostPop)
 	//get books by ID
 	router.GET("categories/:cat_id/book/:book_id", bookService.getBook)
 }
 
-	func initCategoriesRouters(){
-		categoriesService := NewCategoriesService(postgres.CategoryRepoPq{})
-		{
-			// Ensure that the user is logged in by using the middleware
-			router.GET("/categories/0", categoriesService.AllCategories)
-		}
-	}
+func initCategoriesRouters() {
```

Comment:
i cant see where do you call this funciton. its not used. dont you initialize youre routes?

Reply:
Probably it'd be called here. 
router.GET(apiRoute, IndexHandler)
	initUserProfileRouters()
	initBooksRoute()
	initCategoriesRouters(). 

 I created this branch before you merged in changes, so I didn't have this func func Start(port string) as my master wan't updated accordingly. Now a have a whole new banch of conflicts. I believe we'll resolve them in the evening